src/main/java/br/com/soccerbetsurvivor/model/Round.java

```diff
@@ -0,0 +1,43 @@
+package br.com.soccerbetsurvivor.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "soccer_team")
```

Comment:
Why the table name is soccer_team? Why not round?

Reply:
Done